### PR TITLE
Add dedicated testcase for #15881

### PR DESCRIPTION
**Pull request by fhahn**
This PR adds a dedicated regression test for #15881 as @cmr suggested.

### CODE REVIEW

File: src/test/run-pass/issue-15881-model-lexer-dotdotdot.rs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+//
+// ignore-lexer-test FIXME
```
Comment (pczarn) [inline src/test/run-pass/issue-15881-model-lexer-dotdotdot.rs]: this ignore should no longer be here

Comment (fhahn) [inline src/test/run-pass/issue-15881-model-lexer-dotdotdot.rs]: Yup, I've removed the line (and updated the copyright notice date)

**Comment by pczarn** [comment]
Very well.

**Comment by steveklabnik** [comment]
Isn't @cmr away because of studies?

**Comment by fhahn** [comment]
I am not sure if this issue is still relavant after the new model parser got merged, as this issue involves the antlr lexer, which is obselete now, I guess?

**Pull request merged**